src/averageAges.js

```diff
@@ -15,17 +15,25 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const findMale = (element) =>
```

Comment:
Тому що цей колбек повертає булеве значення
```suggestion
  const isMale = (element) =>
```